api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -35,6 +57,8 @@ type VirtualMachineClassHardware struct {
 	Memory  resource.Quantity `json:"memory,omitempty"`
 	// +optional
 	Devices VirtualDevices    `json:"devices,omitempty"`
+	// +optional
+	InstanceStorage InstanceStorage `json:"instanceStorage,omitempty"`
```

Comment:
We agreed that InstanceStorage should be a peer of Devices, not part of it. Please check the internal VMODL change and align with that. Ping Narendra privately for a link to that review. Thanks!

Reply:
My mistake. The GH view on my phone confused me. The current model is aligned with what I was thinking.![image](https://user-images.githubusercontent.com/101085/129519451-fbe94fde-d19d-4359-aae3-71f6c71b6363.png)